mne/channels/montage.py

```diff
@@ -758,6 +760,18 @@ def _backcompat_value(pos, ref_pos):
                 raise ValueError('Cannot use match_case=False as %s channel '
                                  'name(s) require case sensitivity' % n_dup)
 
+        # use lookup table to match unrecognized channel names to known aliases
+        if match_alias:
+            alias_dict = (match_alias if isinstance(match_alias, dict) else
+                          CHANNEL_LOC_ALIASES)
+            if not match_case:
+                alias_dict = {k.lower(): v.lower() for k, v in
+                              alias_dict.items()}
+
+            info_names_use = [alias_dict[use] if use in alias_dict
+                              and alias_dict[use] in ch_pos_use else use
+                              for use in info_names_use]
```

Comment:
@tczhangzhi any way to make this unwieldly list comprehension a little clearer?  (perhaps by using `alias_dict.get(use, use)` in some way?  It might also help readability if your iterator variable has a clearer name (e.g., since the values in `info_names_use` are channel names, maybe the iterator variable could be called `ch_name` rather than `use`?)